tasks/cephfs/fuse_mount.py

```diff
@@ -100,11 +100,16 @@ def list_connections():
         self.fuse_daemon = proc
 
         # Wait for the connection reference to appear in /sys
+        mount_wait = self.client_config.get('mount_wait', 0)
+        if mount_wait > 0:
+            log.info("Fuse mount waits {0} seconds before checking /sys/".format(mount_wait))
+            time.sleep(mount_wait)            
+        timeout = self.client_config.get('mount_timeout', '30')
```

Comment:
I'm not sure how the git history worked out here but see 8e275049a0a8b976fc9eb5c63dcefe01bedcc711